docs/apis-tools/testing/getting-started.md

```diff
@@ -111,8 +113,13 @@ public class MyProcessTest {
     @Autowired private CamundaProcessTestContext processTestContext;
 
     @Test
-    void shouldCompleteProcessInstance() {
-        // given: the processes are deployed
+    void shouldCreateProcessInstance() {
+        // given
+        client
+                .newDeployResourceCommand()
+                .addResourceFromClasspath("my-process.bpmn")
+                .send()
+                .join();
```

Comment:
:wrench: In a Spring test case, it is uncommon to deploy the BPMN process directly. Usually, this is done by the process application via [deployment annotation](https://docs.camunda.io/docs/next/apis-tools/camunda-spring-boot-starter/getting-started/#deploy-process-models).

Reply:
I see, thanks. I changed that to replace it with an annotation. Could you check?